sema/commons/j2/j2_functions.py

```
@@ -145,24 +193,24 @@ def xsd_format(content, type_name: str, quote: str = "'"):
+def uritexpand(template: str, context: dict) -> str:
     return URITemplate(template).expand(context)
 
 
-def regexreplace(find: str, replace: str, text: str):
+def regexreplace(find: str, replace: str, text: str) -> str:
     return re.sub(find, replace, text)
 
 
 class ValueMapper:
     def __init__(self):
         self._map = dict()
```

Ruff (0.8.2)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)